backend_new/prisma/seed-helpers/user-factory.ts

```diff
@@ -1,11 +1,15 @@
 import { Prisma } from '@prisma/client';
+import { randomAdjective, randomNoun } from './word-generator';
 
 export const userFactory = (
   roles?: Prisma.UserRolesUncheckedCreateWithoutUserAccountsInput,
+  firstName?: string,
+  lastName?: string,
+  email?: string,
 ): Prisma.UserAccountsCreateInput => ({
-  email: 'admin@example.com',
-  firstName: 'First',
-  lastName: 'Last',
+  email: email || `${randomNoun()}@${randomAdjective()}.com`,
```

Comment:
If you are pulling out the default email we will need to update the seed-dev.ts and seed-staging.ts to make sure the admin@example.com account is created for testing.

Reply:
curious if you think we should keep the admin@example.com account or if we should create a kinda spread of accounts instead? 

With MFA multiple users trying to use admin@example.com causes some problems so maybe having a few users that are different may be helpful?

Reply:
That's a good point. I think `admin@example.com` should still be used for the dev seeding since theoretically that is just local or testing purposes so there should just be one person logging in. For staging I feel like we should have the admin@example.com as well but then after we seed a new site we manually go in to create accounts. I guess that is what we currently do, so I wonder if there is a better way

Reply:
I think maybe on staging after we use the admin user to create users for a few people we delete the admin user out so that its no longer accessible

the other option is we add  people straight into the seed script, but I'm hesitant putting our emails into a file thats openly accessible on github

I went ahead and have admin@example.com get created on both the dev and staging levels for now at least

Reply:
@YazeedLoonat and @ludtkemorgan , is this conversation resolved?

Reply:
yeah I think the resolution was re-adding the admin@example.com email and then the "how do we use this in staging" question is something we can address when we do the migration work and set up staging on prisma

resolve convo!